deployment/k8sPaiLibrary/template/kubernetes-cleanup.sh.template

```diff
@@ -61,3 +61,5 @@ if [ -f "kubernetes.tar" ]; then
     rm -rf kubernetes.tar
 
 fi
+
+apt-get -y purge ntp
```

Comment:
What if the ntp is installed by others? Why to unstall it?

Reply:
yes, consider this, this is a little like install docker-ce and uninstall docker-ce type problem. If current PAI default not uninstall docker when clean k8s, i will remove this uninstall cmd and let user customized uninstall ntp? What's your suggestion?

Reply:
Don't uninstall it.

Reply:
removed